apps/web-giddh/src/app/multi-currency-reports/profit-loss/profit-loss-report.component.ts

```
@@ -0,0 +1,261 @@
+    /**
+     * Initializes the component with report data from the store.
+     *
+     * @returns {void}
+     * @memberof ProfitLossReportComponent
+     */
+    public ngOnInit(): void {
+        this.componentStore.reportDataList$.pipe(takeUntil(this.destroyed$)).subscribe((response) => {
+            if (response) {
+                let data = prepareProfitLossData(cloneDeep(response)) as ProfitLossData;
+                let cogs;
+                if (data && data.incomeStatement && data.incomeStatement.costOfGoodsSold) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)